lighthouse-extension/gulpfile.js

```diff
@@ -118,7 +118,8 @@ gulp.task('browserify-lighthouse', () => {
       .ignore('source-map')
       .ignore('whatwg-url')
       .ignore('url')
-      .ignore('debug/node');
+      .ignore('debug/node')
+      .ignore('pako/lib/zlib/inflate.js');
```

Comment:
fair. I was just looking at the source tree of pako. I guess the require that get done in browserify is selective already.

:)